roles/openshift_logging/defaults/main.yml

```diff
@@ -27,7 +27,7 @@ openshift_logging_kibana_proxy_debug: false
 openshift_logging_kibana_proxy_cpu_limit: null
 openshift_logging_kibana_proxy_memory_limit: null
 openshift_logging_kibana_replica_count: 1
-openshift_logging_kibana_edge_term_policy: Redirect
+openshift_logging_kibana_edge_term_policy: null
```

Comment:
`None` didn't work for me, I got a message that it is not an allowed value. I will update the readme.